feature_apps/src/main/java/cm/aptoide/pt/feature_apps/presentation/AppsView.kt

```diff
@@ -1,20 +1,127 @@
 package cm.aptoide.pt.feature_apps.presentation
 
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.feature_apps.data.App
+import cm.aptoide.pt.feature_apps.domain.Bundle
+import cm.aptoide.pt.feature_apps.domain.Type
+import java.util.*
+
+@Composable
+fun AppsScreen(viewModel: BundlesViewModel, type: ScreenType) {
+  val bundles: List<Bundle> by viewModel.bundlesList.collectAsState(initial = emptyList())
```

Comment:
should this be "collected"  here ? I think it should be part of the viewmodel state and collected and mapped on the view model